SConstruct

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python
 
-EnsureSConsVersion(3, 0, 0)
+EnsureSConsVersion(3, 1, 2)
```

Comment:
Worth noting, I bumped that version as it seems like SCons 3.0.0 can't be used anymore with modern Python, pip either can't install it, or it has syntax errors.

3.1.2 is what we use for 4.3 and I believe I came to that number by checking intermediate 3.0 -> 3.1.1 versions which were all broken.